TaylorIntegrationTests/ReporterComparator.swift

```diff
@@ -33,14 +33,14 @@ func !=(lhs: [String:AnyObject], rhs: [String:AnyObject]) -> Bool {
 
 class ReporterComparator {
     
-    func compareReporters(_ firstReporterPath: String, secondReporterPath: String) -> Bool {
-        var violations1 = JSONToViolationParser().parseFile(firstReporterPath)
+    func compareReporter(atPath path1: String, withReporterAtPath path2: String) -> Bool {
```

Comment:
This class takes 2 paths to reports and compare them. I'm not interested how it compares the reports, it's his problem. Also, If I would see `compareViolations` function I would think that I should give it an array of `Violation`.